package-lock.json

```diff
@@ -92,7 +92,7 @@
         "npm": "^6 || ^7 || ^8"
       },
       "peerDependencies": {
-        "amazon-chime-sdk-js": "^2.26.0",
+        "amazon-chime-sdk-js": "^3.0.0-beta.0",
```

Comment:
I don't know how to update this but I feel like the  `peerDependencies` should be beta.0 as well.

Reply:
Updated